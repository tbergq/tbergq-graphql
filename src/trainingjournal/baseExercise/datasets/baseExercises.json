{
  "count": 66,
  "baseExercises": [
    {
      "id": "38",
      "name": "Alternerende benkpress med hantler",
      "videoLink": "",
      "muscleGroup": "Chest",
      "description": "Start med begge på sterk arm. En side føres ned mens den andre holdes strak. Annen hver side"
    },
    {
      "id": "10",
      "name": "Arnold Press",
      "videoLink": "https://www.youtube.com/watch?v=6Z15_WdXmVw",
      "muscleGroup": "Chest",
      "description": ""
    },
    {
      "id": "2",
      "name": "Band pulls apart",
      "videoLink": "https://www.youtube.com/watch?v=BVehf-8FAxQ",
      "muscleGroup": "Chest",
      "description": ""
    },
    {
      "id": "7",
      "name": "Benkpress",
      "videoLink": "https://www.youtube.com/watch?v=0k57fpKM2L4",
      "muscleGroup": "Chest",
      "description": ""
    },
    {
      "id": "36",
      "name": "Benkpress m/hantler",
      "videoLink": "https://www.youtube.com/watch?v=xyzh7OpHEq4",
      "muscleGroup": "Chest",
      "description": ""
    },
    {
      "id": "63",
      "name": "Biceps i kabelmaskin",
      "videoLink": "https://www.youtube.com/watch?v=mJ0-SWKKInI",
      "muscleGroup": "Chest",
      "description": ""
    },
    {
      "id": "19",
      "name": "Bulgarsk utfall",
      "videoLink": "https://www.youtube.com/watch?v=zp89Arzfcg4",
      "muscleGroup": "Chest"
    },
    {
      "id": "67",
      "name": "Cable cross",
      "videoLink": "",
      "muscleGroup": "Chest",
      "description": ""
    },
    {
      "id": "66",
      "name": "Cable cross rygg",
      "videoLink": "",
      "muscleGroup": "Chest",
      "description": ""
    },
    {
      "id": "65",
      "name": "Chest press",
      "muscleGroup": "Chest"
    }
  ]
}