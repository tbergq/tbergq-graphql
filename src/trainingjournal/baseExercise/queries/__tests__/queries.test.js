// @flow

import { generateTestsFromFixtures } from '@kiwicom/test-utils';

import executeTestQuery from '../../../../common/services/executeTestQuery';
import baseExercises from '../../datasets/baseExercises.json';

fetch.mockResponses([JSON.stringify(baseExercises)]);

generateTestsFromFixtures(`${__dirname}/__fixtures__`, input =>
  executeTestQuery(input),
);
