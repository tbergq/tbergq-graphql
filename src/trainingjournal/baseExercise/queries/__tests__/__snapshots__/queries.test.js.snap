// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches expected output: BaseExercises.graphql 1`] = `
~~~~~~~~~~ INPUT ~~~~~~~~~~
query baseExercisesQuery {
  baseExercises {
    edges {
      node {
        id
        name
        muscleGroup
      }
    }
  }
}
~~~~~~~~~~ OUTPUT ~~~~~~~~~~
{
  "data": {
    "baseExercises": {
      "edges": [
        {
          "node": {
            "id": "QmFzZUV4ZXJjaXNlOjM4",
            "name": "Alternerende benkpress med hantler",
            "muscleGroup": "Chest"
          }
        },
        {
          "node": {
            "id": "QmFzZUV4ZXJjaXNlOjEw",
            "name": "Arnold Press",
            "muscleGroup": "Chest"
          }
        },
        {
          "node": {
            "id": "QmFzZUV4ZXJjaXNlOjI=",
            "name": "Band pulls apart",
            "muscleGroup": "Chest"
          }
        },
        {
          "node": {
            "id": "QmFzZUV4ZXJjaXNlOjc=",
            "name": "Benkpress",
            "muscleGroup": "Chest"
          }
        },
        {
          "node": {
            "id": "QmFzZUV4ZXJjaXNlOjM2",
            "name": "Benkpress m/hantler",
            "muscleGroup": "Chest"
          }
        },
        {
          "node": {
            "id": "QmFzZUV4ZXJjaXNlOjYz",
            "name": "Biceps i kabelmaskin",
            "muscleGroup": "Chest"
          }
        },
        {
          "node": {
            "id": "QmFzZUV4ZXJjaXNlOjE5",
            "name": "Bulgarsk utfall",
            "muscleGroup": "Chest"
          }
        },
        {
          "node": {
            "id": "QmFzZUV4ZXJjaXNlOjY3",
            "name": "Cable cross",
            "muscleGroup": "Chest"
          }
        },
        {
          "node": {
            "id": "QmFzZUV4ZXJjaXNlOjY2",
            "name": "Cable cross rygg",
            "muscleGroup": "Chest"
          }
        },
        {
          "node": {
            "id": "QmFzZUV4ZXJjaXNlOjY1",
            "name": "Chest press",
            "muscleGroup": "Chest"
          }
        }
      ]
    }
  }
}
`;
